x/nft/blockchain/init.go

```diff
@@ -25,8 +25,8 @@ func (i *Initializer) FromGenesis(opts weave.Options, db weave.KVStore) error {
 }
 
 type genesisBlockchainToken struct {
-	ID    string `json:"id"`
-	Owner string `json:"owner"`
+	ID    weave.Address `json:"id"`
+	Owner weave.Address `json:"owner"`
```

Comment:
Is there a hex -> binary deconding happening magically?

Reply:
Yes :grin: